pkgs/top-level/impure.nix

```diff
@@ -82,7 +90,10 @@ in
 assert args ? localSystem -> !(args ? system || args ? platform);
 
 import ./. (builtins.removeAttrs args [ "system" "platform" ] // {
-  inherit config overlays crossSystem crossOverlays;
+  inherit overlays crossSystem crossOverlays;
+
+  config = defaultConfig // config;
```

Comment:
config can be a function:
https://github.com/NixOS/nixpkgs/blob/71184f8e157672789602d3f28bdd3c8079800687/pkgs/top-level/default.nix#L50-L56
This broke nix-review.

Reply:
thanks for debugging this and sorry about the breakage!